.github/workflows/instrumentations_1.yml

```diff
@@ -39,6 +40,8 @@ jobs:
         exclude:
           - python-version: py311
             package: "prometheus-remote-write"
+          # - python-version: py312
+          #   package: "prometheus-remote-write"
```

Comment:
Unsure about this "exclude section"

Reply:
the test works fine here with python 3.12. Plenty of warnings but tox is happy

```
===================================================== 27 passed, 345 warnings in 0.07s =====================================================
  py312-test-exporter-prometheus-remote-write: OK (17.43=setup[1.48]+cmd[3.12,2.86,3.18,2.97,3.47,0.35] seconds)
```